packages/network/src/solana/transactions.ts

```diff
@@ -52,3 +79,131 @@ export const simulateTransaction = async (
 	const simulatedTx = await connection.simulateTransaction(tx);
 	logger.debug('Simulated transaction:', simulatedTx);
 };
+
+export const constructSendSOLTransaction = async (
+	connection: Connection,
+	sender: PublicKey,
+	receiver: PublicKey,
+	amount: number,
+) => {
+	const instructions = [
+		SystemProgram.transfer({
+			fromPubkey: sender,
+			toPubkey: receiver,
+			lamports: amount,
+		}),
+	];
+
+	const blockhash = (await connection.getLatestBlockhash('finalized'))
+		.blockhash;
+
+	const message = new TransactionMessage({
+		payerKey: new PublicKey(sender),
+		recentBlockhash: blockhash,
+		instructions,
+	}).compileToV0Message();
+
+	return new VersionedTransaction(message);
+};
+
+export const constructSendSPLTokenTransactionInSol = async (
```

Comment:
I prefer to separate network for different network, like solana. We can import by `@walless/network/solana` or using `export * as solana` in the index file of network package, then we will have `import { solana } from `@walless/network` and use like `solana.constructSendSPLTokenTransaction` without `inSol`

Reply:
InSol mean to differentiate token to pay fee between native Sol and SPL token through gasilon